serverless/metering-serverless.adoc

```diff
@@ -0,0 +1,62 @@
+[id="metering-serverless"]
+= Using metering with {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: metering-serverless
+
+As an {product-title} cluster administrator, you can use Metering to analyze what is happening in your {ServerlessProductName} cluster.
+
+.Additional resources
+For more information about Metering on {product-title}, see xref:../metering/metering-about-metering.adoc[About Metering].
+
+[NOTE]
+====
+Some of the following instructions use the `openshift-metering` project and namespace, however for use with {ServerlessProductName}, you should use `metering` as the project and namespace.
+====
+
+[id="installing-metering-serverless_{context}"]
+== Installing Metering
+For information about installing Metering on {product-title}, see xref:../metering/metering-installing-metering.adoc[Installing Metering].
+
+[id="datasources-metering-serverless_{context}"]
+== Applying Datasources for Knative Serving Metering
+The following `ReportDataSources` are available for use with Metering and the {ServerlessProductName} Knative Serving component:
+
+`knative-service-cpu-usage`:: Accumulated CPU seconds used per Knative service over the report time period.
+`knative-service-memory-usage`:: Average Memory consumption per Knative service over the report time period.
+
+These `ReportDataSources` are found in the `datasources` directory.
+You can apply them by using the following command:
+
+```
+$ oc apply -f datasources/
```

Comment:
Where does the datasources directory come from? We should mention that users should checkout certain repo. But maybe it would be better to include the example yaml files directly (not a whole directory).

Reply:
@rhuss would this be the https://github.com/rhuss/knative-reports repo?
Are the two yaml files here the only two expected for now?

Reply:
Used this repo and called out the yaml files for now - please lmk if there's a different repo to point to or any further changes. Thanks.

Reply:
I feel like we should inline them. Or if we don't inline them we should move the repo to somewhere more "official" like the openshift-knative organisation.

Reply:
I must agree with @markusthoemmes , usually the examples are included in our docs directly, not as reference to external github repos.

Reply:
That makes sense, thanks for the input!

Reply:
I think I misunderstood you originally @mgencur sorry about that.
Should I just add a step to say "create X yaml file by copying the sample" and then include the whole file for each yaml?
I'd imagine we need to do the same for `queries` then too :)

Reply:
Hmm, I'd leave out any words about "copying" anything, we can just include every single yaml file in our docs, including `queries` . Overall, there are not too many of them.